fsw/src/eci_app.h

```diff
@@ -26,114 +25,119 @@ extern "C"
+        boolean *StatusFlag;
+    } ECI_Flag_t;
 
-/**@ingroup eci_event
- * Struct to define an EVS event message to be registered.
- */
-typedef struct {
-  /** Event Block describes how many data points  */
-  uint8 eventBlock;
-  /** Event Id  - unique id set by the user*/
-  uint8* eventID;
-  /** Event Type - debug, info, error, crit set by user */
-  uint8* eventType;
-  /** Event Mask - filter set by user */
-  uint32* eventMask;
-   /** Flag indicating simulink event has occurred */
-  boolean* eventFlag;
-  /** Msgpoint to send with an event taken from observable signal */     
-  uint8* eventMsg;
-  /** Location string */
-  char* loc;
-  /** First data point */
-  double* data_1;
-  /** Second data point */
-  double* data_2;
-  /** Third data point */
-  double* data_3;
-  /** Fourth data point */
-  double* data_4;
-  /** Fifth data point */
-  double* data_5;
-} ECI_Evs_t;
+    /**@ingroup eci_event
+     * Struct to define an EVS event message to be registered.
+     */
+    typedef struct
+    {
+        /** Event Block describes how many data points  */
+        uint8 eventBlock;
+        /** Event Id  - unique id set by the user*/
+        uint8 *eventID;
+        /** Event Type - debug, info, error, crit set by user */
+        uint8 *eventType;
+        /** Event Mask - filter set by user */
+        uint32 *eventMask;
+        /** Flag indicating simulink event has occurred */
+        boolean *eventFlag;
+        /** Msgpoint to send with an event taken from observable signal */
+        uint8 *eventMsg;
+        /** Location string */
+        char *loc;
+        /** First data point */
+        double *data_1;
+        /** Second data point */
+        double *data_2;
+        /** Third data point */
+        double *data_3;
+        /** Fourth data point */
+        double *data_4;
+        /** Fifth data point */
+        double *data_5;
+    } ECI_Evs_t;
 
-/**@defgroup eci_table ECI Table
- * @ingroup eci_interface
- * How to define a FSW table
- * @{
- */
-/** Table Interface Structure */
-typedef struct {
-    /** Pointer to table  */
-    void**  tblptr;
-    /** Name of table  */
-    char*   tblname;
-    /** Description of table  */
-    char*   tbldesc;
-    /** Filename of table  */
-    char*   tblfilename;
-    /** Size of table */
-    uint32  tblsize;
-    /** Table validation func */
-    void*   tblvalfunc;   
-} ECI_Tbl_t;
-/**@}*/
+    /**@defgroup eci_table ECI Table
+     * @ingroup eci_interface
+     * How to define a FSW table
+     * @{
+     */
+    /** Table Interface Structure */
+    typedef struct
+    {
+        /** Pointer to table  */
+        void **tblptr;
+        /** Name of table  */
+        char *tblname;
+        /** Description of table  */
+        char *tbldesc;
+        /** Filename of table  */
+        char *tblfilename;
+        /** Size of table */
+        uint32 tblsize;
+        /** Table validation func */
+        void *tblvalfunc;
+    } ECI_Tbl_t;
+    /**@}*/
```

Comment:
Some weird formatting going on here. For some reason, its indenting everything... maybe because of the doxygen comment above for some reason?

Reply:
If we need to tweak the code/comment to prevent this odd formatting, feel free. There were 1-2 other places I saw weird indents like this.

Reply:
Slightly tweaked code to prevent weird formatting, let me know if any other files have something similar.